CONTRIBUTING.md

```diff
@@ -154,29 +168,30 @@ $ git push origin <topic-branch-name>
 
 ### Step 6: Pull Request
 
-Open a Pull Request from your fork of the repository to the `main` branch of the IF repository with a clear title and description according to [template](.github/PULL_REQUEST_TEMPLATE.md).
+Open a Pull Request from your fork of the repository to the `dev` branch of the IF repository with a clear title and description according to [template](.github/PULL_REQUEST_TEMPLATE.md).
 
 Pull requests will not be reviewed unless they pass all CI. This includes a lint check and running our unit tests.
 
 ## Coding guidelines
 
 ### Code structuring patterns
 
-Avoid having functions which are responsible to do multiple things at the same time. Make sure one function/method does one thing, and does it well.
+Avoid having functions which are responsible to do multiple things at the same time. Make sure one function/method does one thing, and does it well. 
 
 ### Functional Programming
 
 We have a preference towards functional programming styles in the IF. This is because it makes it easier for different functions to be developed in isolation, composed in complex ways and executed in parallel.
 
 We recommend starting with these [basic principles and guidelines](https://dev.to/jamesrweb/principles-of-functional-programming-4b7c) for functional programming.
 
+
 ### Naming conventions
 
-We prefer not to use abbreviations of contractions in parameter names.
+We prefer not to use abbreviations of contractions in parameter names. 
 
-Using fully descriptive names makes the code more readable, which in turn helps reviewers and anyone else aiming to understand how the plugin works.
+Using fully descriptive names makes the code more readable, which in turn helps reviewers and anyone else aiming to understand how the plugin works. 
 
-It also helps to avoid ambiguity and naming collisions within and across plugins. Ensure that names clearly and precisely describe the purpose of an element to make its functionality immediately apparent.
+It also helps to avoid ambiguity and naming collisions within and across plugins. Your name should describe what an element does as precisely as practically possible.
```

Comment:
```suggestion
It also helps to avoid ambiguity and naming collisions within and across plugins. Ensure that names clearly and precisely describe the purpose of an element to make its functionality immediately apparent.
```